src/Arcus.Testing.Tests.Integration/Integration/DataFactory/Fixture/TemporaryDataFactoryDataFlow.cs

```diff
@@ -96,6 +134,32 @@ public static async Task<TemporaryDataFactoryDataFlow> CreateWithCsvSinkSourceAs
             return temp;
         }
 
+        /// <summary>
+        /// Creates a DataFlow with a CSV source and sink on an Azure DataFactory resource.
+        /// </summary>
+        public static async Task<TemporaryDataFactoryDataFlow> CreateWithCsvSinkSourceAndDataSetParametersAsync(TestConfig config, ILogger logger, Action<AssertCsvOptions> configureOptions)
```

Comment:
@stijnmoreels added custom methods to add source and sink with parameters... Is this OK?

Reply:
See my comment here: https://github.com/arcus-azure/arcus.testing/pull/228#discussion_r1861543473, if we provide an optional set of options to the existing factory method(s), we don't need a dedicated method with exposing this 'parameter' information, but rather, let the test itself decide if it wants to add parameters.